script/seed.js

```diff
@@ -3,9 +3,9 @@
 const spaceshipNames = [
   'Visage Station', 'Outlander', 'Epitome Base', 'Novis', 'Elyse Colony', 'Tartarus Colony', '  Aeris', 'Escort', 'Nott Terminal', 'Aether Terminal', 'Symbolica Station', 'Nox Terminal', 'Phoenix', 'Ecstasis Colony', 'Warden Terminal', 'Glory Station', 'Guardian Base', 'Hyperion Terminal', 'Heirloom', 'Neo', 'Illume Colony', 'Epitome Terminal', 'Mother Terminal', 'Janus', 'Athena Colony', 'Eir Station', 'Dawn', 'Ecstasis Base', 'Utopis Base', 'Pedigree Colony', 'Exposure Colony', 'Juno Colony', 'Chrono Terminal', 'Olympus', 'Hemera Base', 'Heirloom', 'Prometheus Terminal', 'Guardian Base', 'Angel Station', 'Borealis Colony', 'Azure Colony', 'Mythos Terminal', 'Phoenix Terminal', 'Illume Colony', 'Nebula Colony', 'Symbolica', 'Nox Terminal', 'Fauna Base', 'Lumina', 'Legacy Base', 'Hera Base', 'Saga', 'Novis Station', 'Outlander Base', 'Voyage Terminal', 'Daydream Base', 'Paradox', 'Vulcan Station', 'Chronos Base', 'Prism Colony', 'Ancestor Station', 'Athena Terminal', 'Omega Station', 'Awe', 'Orphan', 'Genesis Colony', 'Rogue Station', 'Terra Station', 'Empyrea Station', 'Terran Station', 'Orphan', 'Vulcan Station', 'Epitome', 'Curator Station', 'Hypnos', 'Nero Colony', 'Voyage Base', 'Gaia Colony', 'Eternity', 'Terminus', 'Apollo Colony', 'Miracle Colony', 'Empyrea', 'Dawn Base', 'Helios Colony', 'Saga Colony', 'Ender Station', 'Dawn Colony', 'Minerva Station', 'Marvel', 'Marvel Colony', 'Awe Base', 'Aeon Station', 'Amazone Station', 'Elysium', 'Voyage Terminal', 'Heritage Colony'
 ];
-const fuelTypes = ['plutonium', 'uranium', 'hydrogen'];
-const sizes = ['small', 'medium', 'large'];
-const specialties = ['exploration', 'mining', 'combat', 'transportation'];
+const fuelTypes = ['Plutonium', 'Uranium', 'Hydrogen'];
```

Comment:
Changing these to capitals might mess with something on the admin products pages, I'll have to take a look. But it shouldn't be a big deal if it needs fixing.